src/features/TaskScreen/TaskScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, Dimensions, StyleSheet} from 'react-native';
+import { TabView, SceneMap } from 'react-native-tab-view';
+
+const ToDoRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#ff4081' }]} />
+);
+
+const InProgressRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const VerifyRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#ff4081' }]} />
+);
+
+const DoneRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+function TaskScreen() {
+    const [index, setIndex] = React.useState(0);
+    const [routes] = React.useState([
+        { key: 'ToDo', title: 'ToDo' },
+        { key: 'InProgress', title: 'In Progress' },
+        { key: 'Verify', title: 'Verify' },
+        { key: 'Done', title: 'Done' },
+    ]);
+
+    const renderScene = SceneMap({
+        ToDo: ToDoRoute,
+        InProgress: InProgressRoute,
+        Verify: VerifyRoute,
+        Done: DoneRoute
+    });
+
+    return (
+        <TabView
+            navigationState={{ index, routes }}
+            renderScene={renderScene}
+            onIndexChange={setIndex}
+            initialLayout={initialLayout}
+        />
+    );
+}
+
+const styles = StyleSheet.create({
```

Comment:
You could create your own `styles.ts` file like in the `HomeScreen` component.

Reply:
Good point